src/__test__/reducers/profile/ProfileReducer.test.js

```diff
@@ -0,0 +1,72 @@
+import profileReducer from '../../../reducers/profileReducer';
+
+
+const initialstate = {
+  profile: '',
+};
+
+const loadingInitialState = {
+  loading: false,
+};
+const profile = {
+  first_name: 'first',
+  last_name: 'last',
+};
+
+let error;
+let retrievedProfile;
+
+describe('profileReducer', () => {
+  it(('it tests the reducer initial state'), () => {
+    const newState = profileReducer(initialstate, {});
+    expect(newState).toEqual(initialstate);
+  });
+
+  it('test that loading occurs', () => {
```

Comment:
It is already clear that this is a test. You do not have to use the word `test`
Please use something like `it('should load...'`.

Reply:
Kindly please, sir

Reply:
Reacted to this please